src/plugins/search/plugin.search.js

```diff
@@ -210,13 +210,14 @@ BookReader.prototype.search = function(term = '', overrides = {}) {
   };
 
   this.trigger('SearchStarted', { term: this.searchTerm, instance: this });
-  return $.ajax({
+  await $.ajax({
```

Comment:
we'll need something like `return processSearchResults(await $.ajax(...))`, because

- Previously, the result of the `ajax` call was passed into `processSearchResults` by `.then` .
- Previously, result of the whole function was a Promise, that itself contained the result of `processSearchResults`, so we want to keep the `return` just in case someone that called `search` expected that!